services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/oauth/OpenIDConnectManagerImpl.java

```diff
@@ -86,7 +93,7 @@ public void setClaimProviders(Map<OIDCClaimName, OIDCClaimProvider> claimProvide
 	 */
 	public static List<OAuthScope> parseScopeString(String s) {
 		if (StringUtils.isEmpty(s)) {
-			return Collections.EMPTY_LIST;
+			return Collections.emptyList();
```

Comment:
Please consider throwing a 4xx status in getAuthenticationRequestDescription() if the client is unverified